pandas/core/frame.py

```diff
@@ -9827,10 +9827,10 @@ def nunique(self, axis: Axis = 0, dropna: bool = True) -> Series:
 
         Examples
         --------
-        >>> df = pd.DataFrame({'A': [1, 2, 3], 'B': [1, 1, 1]})
+        >>> df = pd.DataFrame({'A': [4, 5, 6], 'B': [4, 1, 1]})
```

Comment:
Makes sense, could even use simple strings to make things blindingly obvious

Reply:
I attempted to simplify the explaination string as well.